bench-tps/src/cli.rs

```diff
@@ -353,6 +361,39 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(|arg| is_within_range(arg, 1..))
                 .help("The number of unique destination accounts per transactions 'chunk'. Lower values will result in more transaction conflicts.")
         )
+        .arg(
+            Arg::with_name("bind_address")
+                .long("bind-address")
+                .value_name("HOST")
+                .takes_value(true)
+                .validator(solana_net_utils::is_host)
+                .help("IP address to use with connection cache"),
+        )
+        .arg(
+            Arg::with_name("client_node_id")
+                .long("client-node-id")
+                .value_name("PATH")
+                .takes_value(true)
+                .help("File containing a node id (keypair) to communicate with validators"),
```

Comment:
Could you make this help text more descriptive? Why would I need a keypair to communicate with validators?